artifacts_test.py

```diff
@@ -21,6 +21,13 @@
 STRESS_CMD: str = "/usr/bin/cassandra-stress"
 
 
+BACKENDS = {
+    "aws": ["Ec2Snitch", "Ec2MultiRegionSnitch", "SimpleSnitch"],
```

Comment:
let's remove the SimpleSnitch since we check the image and it should be correctly set to the right Snitch according to the cloud provider-specific image